.github/workflows/upl-build.yml

```diff
@@ -0,0 +1,61 @@
+# @file up-build.yml
+#
+# A workflow that builds UefiPayloadPackage's UPL and upload it's artifacts.
+#
+##
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+name: UPL Build
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ['master']
+
+jobs:
+  build_vs2022:
+    strategy:
+      matrix:
+        os: [windows-latest]
+        python-version: ['3.12']
+        tool-chain: ['VS2022']
+        target: ['DEBUG', 'RELEASE']
```

Comment:
Could we drop these down to _only_ DEBUG or RELEASE (or a combination) so the runners to get triggered is 4 builds instead of 8?

Reply:
Done